src/types/color.js

```diff
@@ -3,6 +3,19 @@
 // Note: we define properties with JSdoc since documentation.js doesn't recognize
 // exported types yet. See https://github.com/documentationjs/documentation/issues/680
 
+/**
+ * @property {number}  AA
+ * @property {number}  AALarge
+ * @property {number}  AAA
+ * @property {number}  AAALarge
+ */
+export type ContrastScores = {|
```

Comment:
Since this is being used as a return type for `getColorContrastScores` each key should return `boolean` as opposed to `number`. This should fix the failing Flow tests in the build.

Reply:
Ah, totally! I'll get this fixed up.